src/components/Callout/InfoCallout.tsx

```diff
@@ -33,7 +33,7 @@ export const InfoCallout: React.FC<
       }
       collapsible={collapse}
     >
-      {children}
+      <div className="mt-0.5">{children}</div>
```

Comment:
Fixes this not being centered:

<img width="769" alt="Screen Shot 2023-05-16 at 3 06 47 pm" src="https://github.com/jbx-protocol/juice-interface/assets/96150256/2403a52e-9a0e-4068-9efa-d0b0bd6cb231">